ansible/playbooks/adhoc/uninstall.yml

```diff
@@ -33,21 +47,46 @@
         - kube-scheduler
         - kube-addons
         - etcd
+        - etcd2
       failed_when: false
 
     - name: Remove packages
       action: "{{ ansible_pkg_mgr }} name={{ item }} state=absent"
-      when: not is_atomic | bool
+      when: not is_coreos and not is_atomic | bool
       with_items:
         - flannel
         - kubernetes-node
         - kubernetes-master
         - etcd
+        - etcd2
```

Comment:
We need to disable the etcd2 service at some point. I figured it didn't hurt to put it in the list of existing services to stop, as opposed to making a separate 'CoreOS' stop service task. Thoughts?